blocks/inline-blocks/inline-image.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+export default {
+	id: 'inline-image',
+	title: __( 'Inline Image' ),
+	type: 'image',
+	icon: 'format-image',
+	render( { url, alt, width }, editor ) {
+		const imgWidth = width > 150 ? 150 : width;
+		const img = `<img style="width:${ imgWidth }px;" src="${ url }" alt="${ alt }" />`;
+
+		editor.insertContent( img );
```

Comment:
Would it be possible to remove the editor dependency here?

Reply:
With that, it would also be cool if the block author could return a React element.

Reply:
I removed the editor dependency as you suggested in https://github.com/WordPress/gutenberg/pull/5794/commits/b3f6218266ac54024ffb62cb70ca1eedb3d699c2. I agree, having the option of returning a React element would be cool -- I'll look into how best to do that.